step1/query.sql

```diff
@@ -55,20 +55,20 @@ ORDER BY
 -- Find the name of all the people that have played in a movie they directed, and order them by their names
 -- (increasing alphabetical order).
 SELECT
-    P.name
+    DISTINCT P.name
 FROM
     person P,
     (
         SELECT
-            person_id
+            person_id,movie_id
```

Comment:
I recommend you to use a plugin called "prettier" in vscode so that it can format this line adding a space after ",". Anyway.